packages/core/src/browser/tree/tree-widget.ts

```diff
@@ -96,8 +107,9 @@ export class TreeWidget extends VirtualWidget implements StatefulWidget {
     protected renderTree(model: ITreeModel): h.Child {
         if (model.root) {
             const props = this.createRootProps(model.root);
-            return this.renderNodes(model.root, props);
+            return this.renderSubTree(model.root, props);
         }
+        // tslint:disable-next-line:no-null-keyword
```

Comment:
why `return null` here? the return type is non-null.

Reply:
From phosphor: `Child = (string | VirtualNode | null) | Array<string | VirtualNode | null>` So yes, it can be `null`.